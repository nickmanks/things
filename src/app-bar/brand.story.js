/* global module */
import React from 'react';
import {storiesOf} from '@storybook/react';
import Brand from './brand';


storiesOf('Simple/Brand', module)
  .add('things logo and name', ()=> (
    <Brand />
  ))
  .add('mobile logo and name', ()=> (
    <Brand mobile={true} />
  ));
