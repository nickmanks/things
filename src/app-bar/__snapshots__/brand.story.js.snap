// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Simple/Brand mobile logo and name 1`] = `
<a className={"app-bar-logo"}>
  <img className={"app-bar-logo-img-mobile"} src={"favicon.png"} />
</a>
`;

exports[`Storyshots Simple/Brand things logo and name 1`] = `
<a className={"app-bar-logo"}>
  <img className={"app-bar-logo-img"} src={"favicon.png"} />
  <h4 className={"app-bar-logo-name"}>
     things 
  </h4>
</a>
`;
